src/api/app/components/add_review_collapsible_component.html.haml

```diff
@@ -0,0 +1,22 @@
+.card.collapse.position-absolute.w-100.mt-2.shadow-lg{ id: 'review_form_collapse' }
```

Comment:
```suggestion
.card.collapse.position-absolute.w-100.mt-2.shadow-lg#review_form_collapse
```

Reply:
rubocop want the lisp style for the id... dont know if the cop makes sense

Reply:
I had a closer look and this is what haml_lint outputs:

> IdNames: `id` attribute must be in lisp-case

Changing `review_form_collapse` to `review-form-collapse` is what needs to happen. I'm handling it.